docs/knowledgebase/integrate/subkey.md

```diff
@@ -258,23 +249,21 @@ Recall the SS58 address from the same seed phrase,
 address.
 
 ```bash
-$ subkey inspect "5CXFinBHRrArHzmC6iYVHSSgY1wMQEdL2AiL6RmSEsFvWezd/joe/polkadot/0"
+$ subkey inspect-key --uri "5CXFinBHRrArHzmC6iYVHSSgY1wMQEdL2AiL6RmSEsFvWezd/joe/polkadot/0"
 Public Key URI `5CXFinBHRrArHzmC6iYVHSSgY1wMQEdL2AiL6RmSEsFvWezd/joe/polkadot/0` is account:
-  Network ID/version: substrate
+  Network ID/version: polkadot
```

Comment:
How did you get this? I installed the RC6 version and it still defaults to Substrate.

Reply:
Here is how I got that:

```shell
~: subkey --version
subkey 2.0.0-rc6
~: subkey inspect-key --uri "5CXFinBHRrArHzmC6iYVHSSgY1wMQEdL2AiL6RmSEsFvWezd/joe/polkadot/0"
Public Key URI `5CXFinBHRrArHzmC6iYVHSSgY1wMQEdL2AiL6RmSEsFvWezd/joe/polkadot/0` is account:
  Network ID/version: polkadot
  Public key (hex):   0x06382e496cb8b1664501bbfcc8205c7b1ddc2402e32ef0479b18abdf326ca342
  Account ID:         0x06382e496cb8b1664501bbfcc8205c7b1ddc2402e32ef0479b18abdf326ca342
  SS58 Address:       199ydvjJEQP8fRrjfvFT6WAA8rhz3sUCdhFnrQzLNueSU5G
```

Reply:
Well isn't that wonderful
```
~> subkey --version
subkey 2.0.0-rc6
```